spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/ZipkinMessageListener.java

```diff
@@ -196,6 +191,13 @@ public Cloud cloud() {
 			return new CloudFactory().getCloud();
 		}
 
+		// TODO: what config shoud we use? zipkin uses zipkin.collector.sample-rate
+		@Bean
+		TraceIdSampler traceIdSampler() {
```

Comment:
How does this correlate with https://github.com/spring-cloud/spring-cloud-sleuth/pull/90/files ?